third_party/move/move-compiler-v2/tests/uninit-use-checker/use_before_assign.exp

```diff
@@ -0,0 +1,29 @@
+============ initial bytecode ================
+
+[variant baseline]
+fun <SELF>_0::main() {
+     var $t0: u64
+     var $t1: u64
+  0: $t1 := infer($t0)
+  1: return ()
+}
+
+
+Diagnostics:
+error: use of unassigned local `x`
+  ┌─ tests/uninit-use-checker/use_before_assign.move:4:9
+  │
+4 │     let y = x;
```

Comment:
This is a bug due to location information not properly propagated to stackless bytecode. I have filed a bug for it: https://github.com/aptos-labs/aptos-core/issues/11599